standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetastoreDefaultTransformer.java

```diff
@@ -558,6 +566,39 @@ public MetastoreDefaultTransformer(IHMSHandler handler) throws HiveMetaException
     return ret;
   }
 
+  static enum TableLocationStrategy {
+    seqsuffix {
+      @Override
+      Path getLocation(IHMSHandler hmsHandler, Database db, Table table, int idx) throws MetaException {
+        if (idx == 0) {
+          return getDefaultPath(hmsHandler, db, table.getTableName());
+        }
+        return getDefaultPath(hmsHandler, db, table.getTableName() + "-" + idx);
```

Comment:
I have a mild concern about this convention. It might be confusing to use a suffix. would a prefix serve us better? with the latter scheme, the original table name can still be preserved?
create table test;
alter table rename test to test1;
create table test; --> test is now at location test-0
create table test-0; --> test-0 is now at location test-1 (would using 0_test-0 be better than using test-1?)